elements/urth-viz-explorer/urth-viz-explorer.html

```diff
@@ -56,6 +56,15 @@
                 notify: true,
                 observer: '_typeChanged'
             },
+
+            /**
+             * Allows user to select multiple chart components (bars in barchart, slices in pie chart, etc).
+             */
+            multiSelect: {
```

Comment:
I need to leave this one here in urth-viz-explorer because urth-viz-explorer doesn't have urth-viz-chart-common-behavior.